src/main/java/com/wimbli/WorldBorder/BorderData.java

```diff
@@ -291,26 +292,33 @@ public Location correctedPosition(Location loc)
 	}
 
 	//these material IDs are acceptable for places to teleport player; breathable blocks and water
-	public static final LinkedHashSet<Integer> safeOpenBlocks = new LinkedHashSet<Integer>(Arrays.asList(
-		 new Integer[] {0, 6, 8, 9, 27, 28, 30, 31, 32, 37, 38, 39, 40, 50, 55, 59, 63, 64, 65, 66, 68, 69, 70, 71, 72, 75, 76, 77, 78, 83, 90, 93, 94, 96, 104, 105, 106, 115, 131, 132, 141, 142, 149, 150, 157, 171}
+	public static final LinkedHashSet<Material> safeOpenBlocks = new LinkedHashSet<Material>(Arrays.asList(
+		 new Material[] {
+				Material.AIR,            Material.OAK_SAPLING,   Material.WATER,      Material.POWERED_RAIL, Material.DETECTOR_RAIL,  Material.COBWEB,
+				Material.TALL_GRASS,     Material.DEAD_BUSH,     Material.DANDELION,  Material.POPPY,        Material.BROWN_MUSHROOM, Material.RED_MUSHROOM,
+				Material.TORCH,          Material.REDSTONE_WIRE, Material.WHEAT,      Material.OAK_SIGN,     Material.OAK_DOOR,       Material.LADDER,
+				Material.OAK_WALL_SIGN,  Material.LEVER,         Material.IRON_DOOR,  Material.RAIL,         Material.WHITE_CARPET,   Material.STONE_PRESSURE_PLATE,
+				Material.REDSTONE_TORCH, Material.STONE_BUTTON,  Material.SNOW,       Material.SUGAR_CANE,   Material.NETHER_PORTAL,  Material.REPEATER,
+				Material.OAK_TRAPDOOR,   Material.PUMPKIN_STEM,  Material.MELON_STEM, Material.VINE,         Material.NETHER_WART,    Material.TRIPWIRE_HOOK,
+				Material.TRIPWIRE,       Material.CARROTS,       Material.POTATOES,   Material.COMPARATOR,   Material.ACTIVATOR_RAIL, Material.OAK_PRESSURE_PLATE}
```

Comment:
Given the use of OAK_xx throughout, let's go ahead and enumerate the rest of the wood types.